CMSIS/Core_A/Source/irq_ctrl_gic.c

```diff
@@ -216,6 +221,8 @@ __WEAK uint32_t IRQ_GetMode (IRQn_ID_t irqn) {
       mode |= IRQ_MODE_TRIG_LEVEL;
     }
 
+    if (val & 1U)
+    	mode |= IRQ_MODE_MODEL_1N;
```

Comment:
Please add curly braces around the block.

Reply:
done